jwql/jwql_monitors/monitor_cron_jobs.py

```diff
@@ -219,27 +223,27 @@ def missing_file_check(avg_time_between, uncertainty, latest_file):
 @log_fail
 @log_info
 def status(production_mode=True):
-    """Main function: determine the status of the instrument montiors by examining
-    log files.
+    """Main function: determine the status of the instrument montiors
+    by examining log files.
 
     Parameters
     ----------
     production_mode : bool
-        If true, look in the main log directory. If false, look in the dev log
-        file directory.
+        If ``True``, look in the main log directory. If ``False``, look
+        in the ``dev`` log file directory.
 
     Returns
     -------
     logfile_status : dict
-        Nested dictionary containing the status for all monitors. Top level keys
-        include all monitors. Within a given monitor, the value is a dictionary
-        containing 'missing_file' and 'status' keys. 'missing_file' is a boolean
-        describing whether or not there is a suspected missing log file based
-        on the timestamps of the existing files. 'status' is a string that is
-        either 'success' or 'failure'.
+        Nested dictionary containing the status for all monitors. Top
+        level keys include all monitors. Within a given monitor, the
+        value is a dictionary containing 'missing_file' and 'status'
+        keys. 'missing_file' is a boolean describing whether or not
+        there is a suspected missing log file based on the timestamps
+        of the existing files. 'status' is a string that is either
+        'success' or 'failure'.
     """
     # Begin logging
-    configure_logging('monitor_cron_jobs', production_mode=production_mode)
```

Comment:
If you take this line out, then how is logging configured in the case where `monitor_cron_jobs` is not called from the command line?

Reply:
@bhilbert4 In that case it wouldn't be configured.  But the problem is, if the script is called from the command line (as it will be in the `cron` job), then `configure_logging` would be called twice, causing two log files to be created for the same run.

I'll have to think about this.  I'm not sure that we would want a new log file created everytime the dashboard webpage is spawned.  That currently goes against our nominal 1-to-1 correspondence between `cron` runs and log files.

Reply:
Alternatively, we could leave `configure_logging()` where you had it, it would still be picked up by the `cron` run.  But having it there goes against our `monitor_template` 🤷‍♂️

Reply:
Well, this is a monitor of monitors, so it's special, right? :) But yeah, I think the bigger issue is that we probably don't want a new log file each time the dashboard page is spawned.